feature/floormap/src/main/java/io/github/droidkaigi/confsched2020/floormap/ui/FloorMapFragment.kt

```diff
@@ -1,17 +1,44 @@
 package io.github.droidkaigi.confsched2020.floormap.ui
 
+import android.os.Bundle
+import android.view.View
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.LifecycleOwner
 import androidx.lifecycle.LiveData
+import androidx.navigation.fragment.navArgs
 import dagger.Module
 import dagger.Provides
 import io.github.droidkaigi.confsched2020.di.Injectable
 import io.github.droidkaigi.confsched2020.di.PageScope
 import io.github.droidkaigi.confsched2020.floormap.R
+import io.github.droidkaigi.confsched2020.floormap.databinding.FragmentFloormapBinding
+import io.github.droidkaigi.confsched2020.model.FloorMap
 
 // TODO: Apply the floor map UI
 class FloorMapFragment : Fragment(R.layout.fragment_floormap), Injectable {
 
+    private val navArgs: FloorMapFragmentArgs by navArgs()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val binding = FragmentFloormapBinding.bind(view)
+
+        binding.floorMapImage.setImageResource(getFloorMapResource())
+    }
+
+    private fun getFloorMapResource(): Int {
+        val floor = FloorMap.fromId(navArgs.roomId) ?: return R.drawable.ic_floor_map
```

Comment:
Thanks!
You can create extension like this 😄 
@DrawableRes
fun FloorMap.drawableResource(): Int